app.py

```diff
@@ -204,17 +212,21 @@ async def create_project_webhook(project_key: str, webhook_url: str, events: lis
     except httpx.HTTPStatusError as e:
         logger.error(
             f"HTTP error when creating webhook for project: {project_key} code: {e.response.status_code} response: {e.response.text}"
-            )
+        )
         return None
 
 async def add_entity_to_port(blueprint_id, entity_object):
     response = await send_port_request(
-        "POST",
-        f"blueprints/{blueprint_id}/entities?upsert=true&merge=true",
-        entity_object
+        method="POST",
+        endpoint=f"blueprints/{blueprint_id}/entities?upsert=true&merge=true",
+        payload=entity_object
     )
     if isinstance(response, dict):
-        logger.error(f'Error adding entity to Port: {response.get("status_code")}, {response.get("response").text}')
+        status_code = response.get("status_code")
+        error_response = response.get("response")
+        logger.error(f"Error adding entity to Port: {status_code}")
+        if error_response:
+            logger.error(f"Error details: {error_response.text}")
```

Comment:
this will create duplication since we're already logging the error message in line 95